util/refresh.go

```diff
@@ -62,3 +65,14 @@ func BuildRefreshCredentialsJSON(refreshToken string, credentialsJSON string) st
 	refreshCredentialsJSON, _ := json.Marshal(refreshCredentials)
 	return string(refreshCredentialsJSON)
 }
+
+// LoadCredentials returns the user specified credentials as a Credentials object.
+// If credentials are not specified by the user, attempt to use application default
+// credentials instead. Returns nil if no credentials can be loaded.
+func LoadCredentials(settings *sgauth.Settings) *credentials.Credentials {
+	creds, err := sgauth.FindJSONCredentials(context.Background(), settings)
+	if err != nil {
+		return nil
```

Comment:
Why is the error discarded here?

Reply:
This function is used in the following place for "attempting" to build a refreshCredential, so we don't really care that it errors out:

refreshCredentialsJSON := BuildRefreshTokenJSON(token.RefreshToken, LoadCredentials(settings))

That said, is it expected that sgauth.FindJSONCredentials should always succeed, and that we should terminate with client error if it doesn't? Let me know if you prefer the latter semantics.

Reply:
In this case I prefer to just call sgauth.FindJSONCredentials inline and have something like:

```
// Errors of locating JSON credentials are ignored because...
cred, _ := sgauth.FindJSONCredentials(...)
```

Reply:
Done.